README.md

```diff
@@ -1 +1,45 @@
-﻿# Todo List App
+
+# AwesomeBooks
+
+This is a wesbite where you can add and remove books.
+
+![screenshot](Screenshottt.png)
+
+This project is my portfolio.
+
+## Built With
+
+- Javascript
+- HTML
+- CSS
+- Wabpack
+
+[Live Demo](https://github.com/zairdon20/Todo.git)
+
+## Getting Started
+
+### install
+
+- run npm start to view the project locally.
+
+To get the content of this project locally you need to run this command in your terminal:
+
+- git clone your https://github.com/zairdon20/Todo.git
+- cd project My-Portfolio
+- open the index.html file to see project running.
+
```

Comment:
- [ ] as the previous CR mentioned please also add the explanation on how to install packages and run the project using npm, this might look something like below:

- clone the project by running `git clone your https://github.com/zairdon20/Todo.git` in your terminal
- go into the project by running `cd Todo`
- Install packages using the command `npm install`
- run the project locally by entering the command `npm start`